denys/Sunshine/S03.01-Exercise-RecyclerView/build.gradle

```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
```

Comment:
Here you could add the different versions as variables.

See "The Top-level Build File" [here](https://developer.android.com/studio/build/index.html)